workflows/build_references.nf

```diff
@@ -4,16 +4,13 @@
 ========================================================================================
 */
 
-// Don't overwrite global params.modules, create a copy instead and use that within the main script.
-def modules = params.modules.clone()
-
-include { ENSEMBL_DOWNLOAD }                from '../modules/local/ensembl/main'                    addParams( options: modules['ensembl_download'] )
-include { ARRIBA_DOWNLOAD }                 from '../modules/local/arriba/download/main'            addParams( options: modules['arriba_download'] )
-include { STAR_GENOMEGENERATE }             from '../modules/local/star/genomegenerate/main'        addParams( options: modules['star_index'] )
-include { ERICSCRIPT_DOWNLOAD }             from '../modules/local/ericscript/download/main'        addParams( options: modules['ericscript_download'] )
-include { FUSIONCATCHER_DOWNLOAD }          from '../modules/local/fusioncatcher/download/main'     addParams( options: modules['fusioncatcher_download'] )
-include { KALLISTO_INDEX as PIZZLY_INDEX }  from '../modules/nf-core/modules/kallisto/index/main'   addParams( options: modules['pizzly_download'] )
-include { STARFUSION_DOWNLOAD }             from '../modules/local/starfusion/download/main'        addParams( options: modules['starfusion_download'] )
+include { ENSEMBL_DOWNLOAD }                from '../modules/local/ensembl/main'
+include { ARRIBA_DOWNLOAD }                 from '../modules/local/arriba/download/main'
+include { STAR_GENOMEGENERATE }             from '../modules/local/star/genomegenerate/main'
```

Comment:
That one could be a nf-core module no?

Reply:
It could be, for the moment however I think we need more tests within this pipeline

Reply:
Yes, I am also up for making local modules for now. They will be very specific for only this pipeline and some will not be changed at all like for example `EricScript` which hasn't been updated in years.

Reply:
Agreed! The ones that only this pipeline will use can stay and should stay local, but everything else could be taken from nf-core/modules no?

Reply:
star/genomegenerate is a nf-core module for example, I think we can take existing modules from nf-core, but keep what is local as local for now